ADD: Description file

zssherman opened the pull request:
Adding roadmap-project-description.rst, which contains release dates, roadmap description, and pyart description.

zssherman committed:
ADD: New project description with roadmap details and also added release dates.

zssherman committed:
UPD: Adding files from ARM-DOE/pyart-roadmap.

scollis commented:
Thanks @Sherman1991 .. I'll take a look soon

scollis commented:
Hey @Sherman1991 : you ended up committing a new version of the excel file.. You need to remove this commit before I can merge. Get @Rumpkie to help you.. and after that @jjhelmus can school you on cherry picking

zssherman committed:
ADD: Added description file #10

This is a redo of the origional commit that included
a modified excel file. The excel file has been removed.

zssherman committed:
Merge branch 'description' of https://github.com/Sherman1991/pyart-roadmap into description
